imgui/core.pyx

```diff
@@ -1935,6 +1935,220 @@ def menu_item(
     return clicked, inout_selected
 
 
+def open_popup(str name):
+    """Open a popup window.
+
+    Marks a popup window as open. Popups are closed when user click outside,
+    or activate a pressable item, or :func:`close_current_popup()` is
+    called within a :func:`begin_popup()`/:func:`end_popup()` block.
+    Popup identifiers are relative to the current ID-stack
+    (so :func:`open_popup` and :func:`begin_popup` needs to be at
+    the same level).
+
+    .. visual-example::
+        :title: Simple popup window
+        :height: 100
+        :width: 220
+        :auto_layout:
+
+        imgui.begin("Example: popup window")
+        show_popup = imgui.button("Open")
+        closable = True
+        if show_popup:
+            imgui.open_popup("Popup Title")
+        popup_opened, popup_visible = imgui.begin_popup_modal(
+            "Popup Title",
+            closable
+        )
+        if popup_opened:
+            imgui.text("Popup contents")
+            imgui.end_popup()
+        imgui.end()
+
+    Args:
+        name (str): label of the modal window.
+
+    .. wraps::
+        void OpenPopup(
+            const char* str_id
+        )
+    """
+    cimgui.OpenPopup(_bytes(name))
+
+
+def begin_popup(str name):
+    """Open a popup window.
+
+    Returns ``True`` if the popup is open and you can start outputting
+    content to it. Only call :func:`end_popup()` if :func:`begin_popup()`
+    returned true.
+
+    For practical example how to use this function, please see documentation
```

Comment:
As mentioned before there should be separate usage example in this docstring. This could be something like:

        imgui.begin("Example: simple popup")

        if imgui.button("select"):
            imgui.open_popup("select-popup")

        imgui.same_line()

        if imgui.begin_popup("select-popup"):
            imgui.text("Select one")
            imgui.separator()
            imgui.selectable("One")
            imgui.selectable("Two")
            imgui.selectable("Three")
            imgui.end_popup()

        imgui.end()

This looks like this: 
![example_snippet](https://cloud.githubusercontent.com/assets/1258054/23613198/aeb68c36-027e-11e7-815d-6896c0f9cd30.png)